x-pack/test/security_api_integration/tests/session_invalidate/invalidate.ts

```diff
@@ -96,6 +96,7 @@ export default function ({ getService }: FtrProviderContext) {
     it('should be able to invalidate all sessions at once', async function () {
       const basicSessionCookie = await loginWithBasic(notSuperuserTestUser);
       const samlSessionCookie = await loginWithSAML();
+      await es.indices.refresh({ index: '.kibana_security_session*' });
```

Comment:
__optional nit:__ it would probably a bit better to have this refresh inside `loginWithBasic` and `loginWithSAML` instead, so that we don't need to worry about it if we add more tests in this file in the future. It's not a big deal though, feel free to ignore.

Reply:
I thought about that, but opted for this only to reduce the number of redundant calls to refresh. If that is not a concern then I agree.

Reply:
I think having redundant refresh calls here is fine.